phing/build.yml

```diff
@@ -72,11 +72,18 @@ git:
     pre-commit: ${blt.root}/scripts/git-hooks
     commit-msg: ${blt.root}/scripts/git-hooks
 
+# Site(s) definition.
 multisite:
-  # The docroot/sites/default directory is used by default.
-  name:
-    - default
-    # - another_site
+  default:
+    aliases:
+      local: self
+      remote: ${project.machine_name}.test
+    uri:
+      local: ${project.machine_name}.local
```

Comment:
This is a lot of config. Why do we need per site AND per env uris? I'd prefer to start with per site uris.